### ISSUE TITLE
Quick build 

### ISSUE THREAD

**Issue by ashthespy** [James-Yu/Atom-LaTeX#29]
Quite often, rather than go through the entire chain of `TEX + BIB + TEX + TEX` just to view a small change, it would be nice to have an option to just have different tool chain say `TEX`. I currently use `arara` so in my main file I set all the directives, so in essence, I could comment out the rest of the directives, and run only the first `TEX` for "draft" compilation, and run the entire chain for the "final" pdf.  

However, I was wondering if others are also interested in something like this? 
Basically a much simpler version of [TexMaker's Quick Build](http://www.xm1math.net/texmaker/pics/quickbuild.png)

**Comment by James-Yu** [comment]
That is exactly what [`latexmk`](http://mg.readthedocs.io/latexmk.html) does. It really does not make sense to re-invent the wheels.

**Comment by ashthespy** [comment]
I think I may not have been to clear ;-)

Currently I use `arara` which as you know, lets you supply per file directives i.e
````
% arara: xelatex: { shell : yes }
% arara: bibtex
% arara: nomencl
% arara: makeindex 
% arara: xelatex: 
% arara: xelatex: { syntex : yes }
````
By setting my toolchain to `arara %DOC -v` I have the final PDF directly by running `atom-latex Build Latex`

But this, takes quite some time on larger documents, and makes reviewing small changes while writing the document or working on images time consuming.  I could overcome this by commenting some of the `arara` directives when working on the document i.e "Draft" mode.

But, it would be nice to have an option for 
* Regular Build: Use normal toolchain - `arara %DOC -v`
* Quick Build: Use quick build toolchain - `xelatex %DOC -synctex=1` 

Anyway, [put together something for myself](fad3d8ea6904a778cdfbd4058f4a2590094a79a4) - but I reckon that this is something extremely particular to my workflow (lots of Tikz ;-))
